docs/pages/enterprise/introduction.mdx

```diff
@@ -7,8 +7,31 @@ h1: Teleport Enterprise
 Teleport Enterprise is a commercial product built around Teleport's open source
 core.
 
-For those that want to jump right in, you can play with the
-[Getting Started Guide for Teleport Enterprise](getting-started.mdx/?scope=enterprise).
+<TileSet>
+<Tile
+title="Get started with Teleport Enterprise"
+href="./getting-started.mdx/?scope=enterprise"
+icon="building"
+>
+
+Learn how to deploy a Teleport Enterprise cluster.
+
+</Tile>
+<ScopedBlock scope={["oss", "cloud"]}>
+<Tile 
+title="Read the docs as a Teleport Enterprise user" 
```

Comment:
```suggestion
title="Read the docs as a self-hosted Teleport Enterprise user" 
```
It's a bit unclear if you're coming here first the diff between Teleport Enterprise and cloud